<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(195, 195, 195)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="boton_start">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: green;
selection-color: green;
selection-background-color: black;</string>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_reiniciar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>710</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: grey;
selection-color: grey;
selection-background-color: black;</string>
    </property>
    <property name="text">
     <string>Reiniciar</string>
    </property>
   </widget>
   <widget class="QLabel" name="nivel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>nivel:</string>
    </property>
   </widget>
   <widget class="QLabel" name="puntaje">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>puntaje:</string>
    </property>
   </widget>
   <widget class="QLabel" name="puntos_valor">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="instrucciones_boton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QLabel" name="nivel_valor">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>49</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_aceptar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>550</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: green;
selection-color: green;
selection-background-color: black;</string>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_denegar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: rgb(244, 67, 54);
selection-color: rgb(244, 67, 54);
selection-background-color: black;</string>
    </property>
    <property name="text">
     <string>Denegar</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="datoInmigrante">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>301</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="reglas">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>291</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
